src/Tribe/Integrations/WPML/Linked_Posts.php

```diff
@@ -73,8 +97,15 @@ public function filter_tribe_events_linked_post_create( $id, $data, $linked_post
 	 *                    WPML is not active or the current language is the default one.
 	 */
 	public function filter_tribe_events_linked_posts_query( $results = null, array $args = array() ) {
+		$cache_key = $this->cache->make_key( func_get_args(), 'filtered_linked_post_query' );
```

Comment:
You cannot use `func_get_args()` directly into a function/method call, you need to save it in a variable first. PHP 5.2 compat.